client/package.json

```diff
@@ -35,5 +35,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "proxy": "http://development-server:80"
```

Comment:
This tells the node development server `"for any requests that aren't for static assets, proxy to this address". More details can be found [here](https://create-react-app.dev/docs/proxying-api-requests-in-development/).

Interestingly, from reading that article , it looks like we don't need to be sending the hostname of the server when using `fetch`, so our `SERVER_URL` fretting may have been for naught.

Reply:
worth us making changes again, or should we just leave it now?

Reply:
Not worth atm, but maybe add to a TODO list.